includes/functions.php

```diff
@@ -2601,6 +2601,13 @@ function tsml_compare_imported_meeting($local_meeting, $import_meeting)
         }
         $normalized_meetings[] = $normalized_meeting;
     }
+
```

Comment:
don't compare group fields when no group present

Reply:
Testing Failed. Using https://[sheets.googleapis.com/v4/spreadsheets/13W4lBuRWKpnHNOC_3wTXI5anLVOkyvMmyn4Wvqx1z3c/values/A1:ZZ?key=AIzaSyAdMoeqxk-EWspUIJDwN7GBSwWSzgL6E5A](https://sheets.googleapis.com/v4/spreadsheets/13W4lBuRWKpnHNOC_3wTXI5anLVOkyvMmyn4Wvqx1z3c/values/A1:ZZ?key=AIzaSyAdMoeqxk-EWspUIJDwN7GBSwWSzgL6E5A) for Puget Sound.